docs/api/start.rst

```diff
@@ -36,7 +36,7 @@ Data formatting
 ---------------
 The following pages will tell you about all the end points and how they work. As a convention all data is presented using the `JSON API <http://jsonapi.org/>`_ standard. A notable exception is the "about" end-point which presents its data slightly differently.
 
-When you submit a request (POST or PUT) with dates in them, format them as ``YYYY-MM-DD``.
+Every request should have hte ``Accept: application/json`` header.  When you submit a request (POST or PUT) with dates in them, format them as ``YYYY-MM-DD``.
```

Comment:
Could you fix this?

Reply:
... How can I mess up a one line commit? Sorry, it's fixed now.